cloudigrade/account/tests/helper.py

```diff
@@ -41,6 +42,29 @@ def generate_aws_account(arn=None, aws_account_id=None, user=None, name=None):
     )
 
 
+def generate_account(user=None, name=None):
```

Comment:
Although it's _technically_ allowed in the current state of the code, realistically nothing should be creating barren `Account` objects without also creating a cloud-specific account object (e.g. `AwsAccount`) that references/extends it, and when we add new cloud providers, they will fall under the same rule and make their own cloud-specific account objects (e.g. `AzureAccount`) to do the same.

`Account` isn't strictly abstract (maybe it _should_ be, but I suspect there were ORM/framework reasons for this that I don't recall immediately), but conceptually that's how we should treat it and the other "base" classes (`Instance`, `InstanceEvent`, and `MachineImage`).

Unless there's a strong reason for divorcing the base object from its cloud-specific object, I think we should be using the existing `generate_aws_account` helper instead of this.